packages/button/src/Button.js

```diff
@@ -60,28 +69,23 @@ class Button extends Component {
     );
   }
 
-  mouseOver() {
+  mouseOver = function() {
```

Comment:
Any reasons to change `mouseOver()` to `mouveOver = function()`?
Personally i prefer `mouseOver()`